src/main/kotlin/com/manimdsl/stylesheet/Stylesheet.kt

```diff
@@ -99,23 +102,22 @@ class Stylesheet(private val stylesheetPath: String?, private val symbolTableVis
     }
 
     fun getStyle(identifier: String, value: ExecValue): StyleProperties {
+
         val dataStructureStyle =
-            stylesheet.dataStructures.getOrDefault(value.toString(), StyleProperties()) merge StyleProperties(
-                borderColor = "BLUE",
-                textColor = "WHITE",
-                animate = DefaultAnimationProperties()
-            )
+            stylesheet.dataStructures.getOrDefault(value.name, StyleProperties())
         val style = stylesheet.variables.getOrDefault(identifier, dataStructureStyle)
 
-        return style merge dataStructureStyle
+        val styleProperties = style merge dataStructureStyle
+        return styleProperties
     }
 
     fun getAnimatedStyle(identifier: String, value: ExecValue): AnimationProperties? {
         val dataStructureStyle =
             stylesheet.dataStructures.getOrDefault(value.toString(), StyleProperties()) merge StyleProperties(
```

Comment:
I found the problem - I think you just missed changing this to `value.name`. If you change this and also change the test it should give the expected behaviour